### ISSUE TITLE
9.0 hr_expense: crash when trying to change the "Expense Journal"

### ISSUE THREAD

**Issue by alexis-via** [odoo/odoo#8541]
Here are the steps to reproduce the bug:
1) connect as admin to runbot 9.0
2) Go to the "Expense" Menu
3) Click on "Create", then try to change the value of the field "Expense Journal" -> you get a crash:

```
Error: Expected ")", got "]"

http://101778-9-0-f29d81.runbot6.odoo.com/web/content/363-11d0e23/web.assets_backend.js:2736
Traceback:
advance@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/363-11d0e23/web.assets_backend.js:2736:1
@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/363-11d0e23/web.assets_backend.js:2717:1
expression@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/363-11d0e23/web.assets_backend.js:2734:144
@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/363-11d0e23/web.assets_backend.js:2720:17
expression@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/363-11d0e23/web.assets_backend.js:2734:144
py.parse@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/363-11d0e23/web.assets_backend.js:2851:462
py.eval@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/363-11d0e23/web.assets_backend.js:2866:293
eval_domains/<@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/363-11d0e23/web.assets_backend.js:3005:120
_.forEach@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/455-dbfae5c/web.assets_common.js:12:558
_.mixin/</_.prototype[name]@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/455-dbfae5c/web.assets_common.js:69:521
eval_domains@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/363-11d0e23/web.assets_backend.js:3004:136
eval_domains/<@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/363-11d0e23/web.assets_backend.js:3005:301
_.forEach@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/455-dbfae5c/web.assets_common.js:12:558
_.mixin/</_.prototype[name]@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/455-dbfae5c/web.assets_common.js:69:521
eval_domains@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/363-11d0e23/web.assets_backend.js:3004:136
eval_domains/<@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/363-11d0e23/web.assets_backend.js:3005:301
_.forEach@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/455-dbfae5c/web.assets_common.js:12:558
_.mixin/</_.prototype[name]@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/455-dbfae5c/web.assets_common.js:69:521
eval_domains@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/363-11d0e23/web.assets_backend.js:3004:136
pyeval@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/363-11d0e23/web.assets_backend.js:3015:24
eval_arg@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/363-11d0e23/web.assets_backend.js:3018:61
ensure_evaluated@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/363-11d0e23/web.assets_backend.js:3021:11
.call@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/363-11d0e23/web.assets_backend.js:2888:1
DataSet<.name_search@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/363-11d0e23/web.assets_backend.js:3142:1038
CompletionFieldMixin.get_search_result@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/363-11d0e23/web.assets_backend.js:3557:25
FieldMany2One<.render_editable/<.source@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/363-11d0e23/web.assets_backend.js:3663:103
._search@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/455-dbfae5c/web.assets_common.js:1588:142
$.widget/</proxiedPrototype[prop]</<@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/455-dbfae5c/web.assets_common.js:1124:328
.search@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/455-dbfae5c/web.assets_common.js:1588:8
$.widget/</proxiedPrototype[prop]</<@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/455-dbfae5c/web.assets_common.js:1124:328
$.widget.bridge/$.fn[name]/<@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/455-dbfae5c/web.assets_common.js:1129:13
.each@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/455-dbfae5c/web.assets_common.js:473:758
jQuery.prototype.each@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/455-dbfae5c/web.assets_common.js:461:636
$.widget.bridge/$.fn[name]@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/455-dbfae5c/web.assets_common.js:1126:342
FieldMany2One<.render_editable/<@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/363-11d0e23/web.assets_backend.js:3658:1013
jQuery.event.dispatch@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/455-dbfae5c/web.assets_common.js:740:378
jQuery.event.add/elemData.handle@http://101778-9-0-f29d81.runbot6.odoo.com/web/content/455-dbfae5c/web.assets_common.js:713:151

```

**Comment by alexis-via** [comment]
This is just a typo in the domain of the "journal_id" field in the hr_expense form view.

**Comment by antonylesuisse** [comment]
Fix pushed in 1d6c92a8c65a62fb9024ec5739098496822f6cee
